tests/packages/test_locker.py

```diff
@@ -81,7 +81,9 @@ def test_lock_file_data_is_ordered(locker: Locker, root: ProjectPackage):
     with locker.lock.open(encoding="utf-8") as f:
         content = f.read()
 
-    expected = """\
+    expected = f"""\
+# {GENERATED_COMMENT}
```

Comment:
Looks like flake8 isn't happy with the explicit `#` here -- not sure off the top of my head how to resolve it, but we'll need to solve that to merge this.

Reply:
fixed